README.md

```diff
@@ -1,7 +1,7 @@
-# The Rust Programming Language
+# The Rust Programming Language 
 
 This is the main source code repository for [Rust]. It contains the compiler,
-standard library, and documentation.
+standard library and documentation.
```

Comment:
heh, I prefer the Good Ol' Oxford comma, but I'm happy either way.

Reply:
we generally do use the oxford comma

Reply:
okay, won't change the oxford ways then :)